chart/kubeapps/README.md

```diff
@@ -275,6 +276,12 @@ You could alternatively ensure that the `imagePullSecret` is available in all na
 
 Yes! You can get started by following the [operators documentation](https://github.com/kubeapps/kubeapps/blob/master/docs/user/operators.md).
 
+### Slow response when listing namespaces
+
+Kubeapps uses the logged user account to retrieve the list of namespaces. If the user doesn't have permission to list namespaces, the fallback mechanism verifies if the user has permissions to get secrets for each namespace (to verify it it should be allowed to use that namespace or not). This can lead to a slow response if the number of namespaces of the cluster is big enough.
```

Comment:
```suggestion
Kubeapps uses the currently logged-in user credential to retrieve the list of all namespaces. If the user doesn't have permission to list namespaces, the backend will try again with its own service account to list all namespaces and then iterate through each namespace to check if the user has permissions to get secrets for each namespace (to verify if they should be allowed to use that namespace or not and hence whether it is included in the selector). This can lead to a slow response if the number of namespaces on the cluster is large.
```